### DISCUSSION TITLE
Unmanaged Network Interfaces

### DISCUSSION THREAD

**Discussion by PythonFiend** [telekom-security/tpotce#1137] [Q&A]
- What version of the OS are you currently using `lsb_release -a` and `uname -a`? Debian GNU/Linux 11 (bullseye)
- What T-Pot version are you currently using? Latest
- What edition (Standard, Nextgen, etc.) of T-Pot are you running? Standard
- What architecture are you running on (i.e. hardware, cloud, VM, etc.)? vSphere exsi 6.7
- Did you have any problems during the install? If yes, please attach `/install.log` `/install.err`. No
- How long has your installation been running? Fresh install
- Did you install upgrades, packages or use the update script? No
- Did you modify any scripts or configs? If yes, please attach the changes. No
- Please provide a screenshot of `glances` and `htop`.
- How much free disk space is available (`df -h`)? 256GB
- What is the current container status (`dps.sh`)? All running
- What is the status of the T-Pot service (`systemctl status tpot`)? Active running
- What ports are being occupied? Stop T-Pot `systemctl stop tpot` and run `netstat -tulpen`
- 
![image](https://user-images.githubusercontent.com/75555974/179248486-b4225d59-6e99-4471-b8e9-3d3dd1a226ac.png)

I'm runnign tpot in a sandboxed envirnment wiht no access to the internet. I'm using eth0 as the network interface, but my question is why do I see other interfaces as active and transmitting data as well?

![1](https://user-images.githubusercontent.com/75555974/179248830-5116129f-126b-48e7-afb2-ffa294149c1f.PNG)

Thanks for your help.

**Comment by t3chn0m4g3** [comment]
The `veth` interfaces are docker based and necessary for the docker containers to have network connectivity.
